src/spootnik/reporter.clj

```diff
@@ -246,19 +246,10 @@
       (tmr/stop (tmr/timer registry (->alias alias)))))
   SentrySink
   (capture! [this e]
-    (error e "captured exception")
-    (when raven
-      (try
-        (raven/capture! {:http raven} (:dsn sentry) e {})
-        (catch Exception e
-          (error e "could not send capture")))))
+    (capture! this e {}))
   (capture! [this e tags]
-    (error e "captured exception")
-    (when raven
-      (try
-        (raven/capture! {:http raven} (:dsn sentry) e tags)
-        (catch Exception e
-          (error e "could not send capture")))))
+    (when-let [event-id (raven/capture! {:http_client raven} (:dsn sentry) e tags)] raven
+              (error e (str "captured exception as sentry event: " event-id))))
```

Comment:
you'll lost a raven :bird:

Reply:
Oops! It looks like when-let does not do what I thought it did.

Thanks! :+1: 

Done.